### PR TITLE
Description&format text

### PR DESCRIPTION
**Pull request by BehnamAxo**

### CODE REVIEW

File: axoSlackBot.js [hunk 1]
```diff
@@ -114,15 +114,10 @@ controller.on('rtm_close',function(bot) {
     // });
 });
 
-controller.hears('(get my|get) (.*)(items)(.*)',['direct_message,direct_mention,mention'],function(bot, message){
+controller.hears('(get my|get) (.*)(items)(.*)',['direct_message,direct_mention,mention,ambient'],function(bot, message){
```
Comment (BrettGAxosoft) [inline axoSlackBot.js]: Should only be ambient for axo #

File: axoSlackBot.js [hunk 2]
```diff
@@ -277,7 +272,7 @@ controller.hears('(.*)(axo)(d|f|t|i|[]{0})(\\s|[]{0})(\\d+)(.*)',['direct_messag
        });
 });
 
-controller.hears(['help','Help','HELP'],['direct_message,direct_mention,mention'],function(bot, message){
+controller.hears(['help','Help','HELP'],['direct_message,direct_mention,mention,ambient'],function(bot, message){
```
Comment (BrettGAxosoft) [inline axoSlackBot.js]: keep as original

Comment (BehnamAxo) [reply axoSlackBot.js]: @BrettGAxosoft done

File: axoSlackBot.js (hunk 1 above)
Comment (BehnamAxo) [reply axoSlackBot.js]: done

**Pull request merged**